app/src/main/java/org/simple/clinic/summary/PatientSummaryEvent.kt

```diff
@@ -69,3 +70,5 @@ object ContactDoctorClicked : PatientSummaryEvent() {
 data class FetchedTeleconsultationInfo(val teleconsultInfo: TeleconsultInfo) : PatientSummaryEvent()
 
 object RetryFetchTeleconsultInfo : PatientSummaryEvent()
+
+data class UserLoggedInStatusLoaded(val loggedInStatus: User.LoggedInStatus?) : PatientSummaryEvent()
```

Comment:
`User#loggedInUserImmediate` returns a nullable in the effect handler, so I had to make it nullable in the event as well. Since we are guaranteed to have a user, I will force unwrap it.